setup.cfg

```diff
@@ -35,6 +35,7 @@ project_urls =
 [options]
 packages = find:
 install_requires =
+    qtpy
```

Comment:
add minimal version requirement, Without this update of superqt will not force update of qtpy, so compatibility with PySide6/PyQt6 will be missed.

Reply:
Hmmm, I hesitate to do that in case someone wanted to use an older version of qtpy with qt 5.15?

Its not that superqt required 2.0+, it's that PyQt6 does... thoughts on that?  So it could make some environment resolutions more difficult if we do that.

Thoughts?

Reply:
is there release from the time of introduction qtcompat? If yes hen someone may have code that uses `superqt`,   and have an older version of `qtpy` in the environment, then after the update, the code will be broken. But maybe this is enough small that put information in readme will be enough.